proxy-init/Dockerfile

```diff
@@ -1,8 +1,8 @@
 ## compile proxy-init utility
-FROM gcr.io/linkerd-io/go-deps:817f8e11 as golang
+FROM gcr.io/linkerd-io/go-deps:baf323e4 as golang
 WORKDIR /go/src/github.com/linkerd/linkerd2
 COPY ./proxy-init ./proxy-init
-RUN CGO_ENABLED=0 GOOS=linux go install -v -installsuffix cgo ./proxy-init/
+RUN CGO_ENABLED=0 GOOS=linux go install -v cgo ./proxy-init/
```

Comment:
This is an invalid build command -- I think you also need to remove " cgo".

Reply:
> This is an invalid build command -- I think you also need to remove " cgo".

Oops! Thanks for noticing that. I fixed it.